Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -123,11 +121,25 @@ public func verifySnapshot<A, B>(
         return "Recorded snapshot: …\n\n\"\(snapshotFileUrl.path)\""
       }
 
-      let data = try Data(contentsOf: snapshotFileUrl)
-      let reference = snapshotting.diffing.fromData(data)
+      var allVariantsToTest = [snapshotFileUrl]
+      for c in "abcdefghijklmnopqrstuvwxyz" {
+        let url = snapshotUrl(directoryUrl: snapshotDirectoryUrl, testName: testName, identifier: identifier, pathExtension: snapshotting.pathExtension, variant: String(c))
+        if fileManager.fileExists(atPath: url.path) {
+          allVariantsToTest.append(url)
+        } else {
+          break
+        }
+      }
+
+      var (failure, attachments): (String, [Attachment]) = ("", [])
+      for variantUrl in allVariantsToTest {
+        let data = try Data(contentsOf: variantUrl)
+        let reference = snapshotting.diffing.fromData(data)
 
-      guard let (failure, attachments) = snapshotting.diffing.diff(reference, diffable) else {
-        return nil
+        guard let (f, a) = snapshotting.diffing.diff(reference, diffable) else {
+          return nil
+        }
+        (failure, attachments) = (f, a)
```

Comment:
Likewise, this block feels a bit clumsy but if I try and map here it's rather hard or convoluted to break out of it.